tests/test_currency_network_registry.py

```diff
@@ -92,6 +92,66 @@ def test_add_network_multiple(
     )
 
 
+def test_add_network_twice_does_not_change_metadata(
+    initialized_currency_network_registry_contract, currency_network_contract, accounts
+):
+    """Adding the same network again does not change the initial information.
+
+    Each new registration does throw an event with the current
+    registrar, but the stored metadata from the first call do not change
```

Comment:
Each new registration emits an event with the new registrar, but the original registrar from the first call does not change.

Reply:
:heavy_check_mark: